releases/handbook.md

```diff
@@ -9,22 +9,21 @@ The purpose of this document is to define the roles and processes for the Kubefl
 ### Release Manager
 The Release Manager will be responsible for coordinating the release and taking ultimate accountability for all release tasks to be completed on time.
 
-If multiple Release Team members meet the prerequisites to become a Release Manager, the one from a different organization than the one that drove the previous release should be preferred.
+The previous Release Manager is responsible for choosing their successor that meets the prerequisite. If multiple Release Team members meet the prerequisites to become a Release Manager, the one from a different organization than the one that drove the previous release should be preferred.
 
 **Prerequisites:**
-1. Be a contributor for at least 1 year
-2. Have shadowed in a previous release or be a Release Team member in the past
-3. Contributions in at least 2 Working Groups
+1. Have contributed as a release team member in two or more previous releases
 
-**Time Commitments:** This role will need to be synchronous. The Release Manager is expected to be joining all the weekly and burndown meetings as well as coordinate with the Release Team and the Working Group leads. For the Development phase an estimate of the work needed is 4-8 hours per week. For the next phases the Release Manager will need to devote more time and coordinate the tasks of the release.
+**Time Commitments:** This role will need to be synchronous. The Release Manager is expected to join most of the weekly meetings and coordinate with the Release Team and the Working Group leads. For the Development phase, an estimate of the work needed is 4-8 hours per week. For the next phases, the Release Manager is required to devote more time and coordinate the tasks of the release.
```

Comment:
@jbottum found the sentence you were referring to and modified to "most" instead of "all"

Reply:
yes